src/topmenu/TopMenu.less

```diff
@@ -1,5 +1,42 @@
-@media print {
+.top-menu {
+    position: relative;
+
     .VeilederHeaderMeny {
-        display: none !important;
+        width: 100%;
+    }
+
+    &__nyheter {
+        position: absolute;
+        bottom: 0.25rem;
+        right: 2rem;
+
+        .nytt__knapp svg {
+            filter: invert() brightness(120%);
+        }
+
+        // Hack for luft mellom kant og popup
```

Comment:
Hacky hack. Men dette er tilsynelatende eneste måten for å få luft mellom kant og popup.